pyproject.toml

```diff
@@ -1,123 +1,194 @@
+
+[dependency-groups]
+dev = [
+  "deptry>=0.23",
+  "mkdocs-awesome-nav>=3.1.2",
+  "mkdocs-material>=9.6",
+  "mkdocs>=1.6",
+  "mkdocstrings[python]>=0.30",
+  "mypy>=1.17",
+  "pre-commit>=3.8",
+  "pytest-cov>=4.1",
+  "pytest>=7.4",
+  "ruff>=0.12",
+  "tox-uv>=1.28",
+  "vcrpy"
+]
+
 [project]
-name = "bioutils"
 authors = [
-  { name="biocommons contributors", email="biocommons-dev@googlegroups.com" },
+  {email = "biocommons-dev@googlegroups.com", name = "biocommons contributors"}
 ]
-description = "miscellaneous simple bioinformatics utilities and lookup tables"
-readme = "README.rst"
-license = { file="LICENSE.txt" }
-requires-python = ">=3.10"
 classifiers = [
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "License :: OSI Approved :: Apache Software License",
-    "Operating System :: OS Independent",
-    "Topic :: Scientific/Engineering :: Bio-Informatics",
-    "Topic :: Scientific/Engineering :: Medical Science Apps."
+  "Intended Audience :: Developers",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python",
+  "Topic :: Software Development :: Libraries :: Python Modules"
 ]
-keywords = [
-    "biocommons",
-    "bioinformatics",
-    "genomics",
-    "HGVS",
-    "variation"
-]
-dynamic = ["version"]
 dependencies = [
-    "attrs",
-    "requests",
-]
-
-[project.optional-dependencies]
-dev = [
-    "bandit~=1.7",
-    "build~=0.8",
-    "flake8~=4.0",
-    "ipython~=8.4",
-    "isort~=5.10",
-    "pylint~=2.14",
-    "ruff == 0.4.4",
+  "attrs",
+  "requests"
 ]
-test = [
-    "pytest~=7.1",
-    "pytest-cov~=4.0",
-    "tox~=3.25",
-    "vcrpy",
+description = "miscellaneous simple bioinformatics utilities and lookup tables"
+dynamic = ["version"]
+keywords = [
+  "HGVS",
+  "biocommons",
+  "bioinformatics",
+  "genomics",
+  "variation"
 ]
-docs = ["mkdocs"]
+license = "Apache-2.0"
+name = "bioutils"
+readme = "README.md"
+requires-python = ">=3.10"
 
 [project.urls]
-"Homepage" = "https://github.com/biocommons/bioutils"
-"Bug Tracker" = "https://github.com/biocommons/bioutils/issues"
+Documentation = "https://biocommons.github.io/python-package/"
+Homepage = "https://github.com/biocommons/bioutils"
+Issues = "https://github.com/biocommons/bioutils/issues"
+Repository = "https://github.com/biocommons/bioutils"
 
-[build-system]
-requires = ["setuptools >= 70.1", "setuptools_scm[toml] ~= 8.1"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools.package-data]
-"*" = ["_data/*/*.json.gz"]
+[tool.coverage.report]
+exclude_lines = [
+  # Don't complain about missing debug-only code:
+  "def __repr__",
+  # Don't complain if non-runnable code isn't run:
+  "if __name__ == .__main__.:",
+  "if self.debug",
```

Comment:
`toml-sort` got you here -- this was previously included under the "missing debug-only code" block